GOVERNANCE.md

```diff
@@ -1,18 +1,97 @@
-We decide if a codebase is ready for community co-development based on this document.
+# Governance.md
 
-The standard is maintained by Foundation for Public Code staff.
+The Standard for Public Code is a community governed project.
 
-[We welcome contributions, such as suggestions for changes or general feedback, from anyone.](/CONTRIBUTING.md)
+## Principles
 
-Because of the important role that the Standard for Public Code has in our core process we require the highest standards from the Standard.
+The Standard for Public Code community adheres to the following principles:
 
-We will try to respond promptly to all pull requests.
-The pull request is an opportunity to work together to improve our methods and the Standard.
-We may not accept all changes, but we will explain our logic.
+* Truly open - as little restrictions as possible for anyone to adapt the Standard for Public Code to their context
+* Welcoming and respectful - as a community we want to make it easier for new users to become contributors.
+* Transparent and accessible, changes to the governance, Standard for Public Code code repository, and Standard for Public Code related activities (e.g. level, involvement, etc) are done in public
+* Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles
+
+## Steering team
+
+The community of Standard for Public Code has one steering team.
+
+### Composition
+
+Any active contributor in the community can request to become a steering team member by asking the steering team. The steering team will vote on it (see voting below).
+
+The current team members are:
+
+* Claus Mullie
+* Johan Groenen (Tiltshift, Code for NL)
+
+Ideally, no one organization will employ a majority of the steering team.
+
+### Responsibilities
+
+The steering team members are active contributors who are on a day-to-day basis responsible for:
+
+* Merging pull requests
+* Handling code of conduct violations
+
+Besides the day-to-day activities, the steering team has the joint responsibility to:
+
+* Provide technical direction for the codebase
+* Maintain a roadmap, and contributing principles
+* Resolve issues in development or conflicts between contributors
+* Managing and planning releases
+* Controlling access rights to Standard for Public Code assets such as source repositories, hosting and project calendars
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance as needed
+* Making codebase level decisions
+* Managing the Standard for Public Code brand
+* Licensing and intellectual property changes
+
+### Meetings
+
+The steering team meets regularly.
+Their agenda includes review of the roadmap and issues that are at an impasse.
+The intention of the agenda is not to review or approve all patches.
+(That is mainly being done through the process described in [CONTRIBUTING.md](CONTRIBUTING.md).)
```

Comment:
```suggestion
The intention of the agenda is not to review or approve all patches, as that is mainly done through the process described in [CONTRIBUTING.md](CONTRIBUTING.md).
```

Reply:
```suggestion
The intention of the agenda is not to review or approve patches.
That is done through the process described in [CONTRIBUTING.md](CONTRIBUTING.md).
```
Would this be a bit clearer?

Reply:
From my perspective, not really. “That” in your suggestion is not clearly referring to “review or approve patches”.

Reply:
```suggestion
The intention of the agenda is not to review or approve all patches.
(Reviewing and approving patches is done through the process described in [CONTRIBUTING.md](CONTRIBUTING.md).)
```

Reply:
Have a preference to keep sentences sort and simple. I think this is clear enough.